charts/aws-ebs-csi-driver/values.yaml

```diff
@@ -89,6 +89,17 @@ sidecars:
     securityContext:
       readOnlyRootFilesystem: true
       allowPrivilegeEscalation: false
+  healthMonitor:
```

Comment:
We should consider allowing users to specify their own monitoring interval via the `monitor-interval` flag: https://github.com/kubernetes-csi/external-health-monitor/blob/f11edf7690317fbad79ac0506a166eef3d78bf18/cmd/csi-external-health-monitor-controller/main.go#L55

Reply:
Leaving this open until we decide what the default for `monitor-interval` will be.

Reply:
Every configuration option represents a failure of the software to do the right thing automatically. Every configuration option needs to be documented and protected by unit tests, thereby increasing the cognitive load of user and developer alike. Sometimes they are necessary, but only as a last resort.